File: notebooks/run_config_project.py
# set environment
from pathlib import Path
import sys

# Find project root based on this file's location
_THIS_FILE = Path(__file__).resolve()
FD_NBK     = _THIS_FILE.parent  # notebook folder
FD_PRJ     = FD_NBK.parent      # project root dir
FD_EXE     = FD_PRJ / "scripts" # script folder

# Put scripts on sys.path if not already there
spath = str(FD_EXE)
if spath not in sys.path:
    sys.path.insert(0, spath)

# Import the config
from config_project import *

File: notebooks/run_config_project.R
### get project script
FD_REPO = "/hpc/group/igvf/kk319/repo"
FD_PRJ  = file.path(FD_REPO, "Proj_IGVF_BlueSTARR")
FD_EXE  = file.path(FD_PRJ,  "scripts")

### source
txt_fpath = file.path(FD_EXE, "config_project.R")
source(txt_fpath)